.github/workflows/create-ocp-project.yaml

```diff
@@ -0,0 +1,109 @@
+name: MTA v7.x Workflow end to end tests 
+
+on:
+  workflow_dispatch:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - 'charts/create-ocp-project/**'
+      - .github/workflows/create-ocp-project.yaml
+
+jobs:
+  run-e2e:
+    runs-on: ubuntu-24.04
+    needs: build
+    steps:
+      - name: Create k8s Kind Cluster
+        uses: helm/kind-action@v1.10.0
+        with:
+          cluster_name: kind
+
+      - name: Install Operators Support
+        run: |
+          kubectl apply -f https://raw.githubusercontent.com/operator-framework/operator-lifecycle-manager/master/deploy/upstream/quickstart/crds.yaml
+          # give the apiserver time
+          sleep 5s 
+          kubectl apply -f https://raw.githubusercontent.com/operator-framework/operator-lifecycle-manager/master/deploy/upstream/quickstart/olm.yaml
+
+      - name: Deploy sonataflow-operator
+        run: |
+          helm repo add orchestrator https://parodos-dev.github.io/orchestrator-helm-chart
+          helm install orchestrator orchestrator/orchestrator-k8s
+
+          kubectl get sfp -A
+          kubectl wait --for=condition=Ready=true pods -l "app.kubernetes.io/name=backstage" --timeout=10m
+          kubectl get pods -o wide
+          kubectl wait --for=condition=Ready=true pods -l "app=sonataflow-platform" --timeout=180s
+
+      - name: Deploy MTA serverless workflow and its components
+        run: |
+          TARGET_NS=sonataflow-infra
+          cd charts
+          helm install create-ocp-project create-ocp-project -n "${TARGET_NS}"
```

Comment:
I'm not sure this workflow can be tested on k8s.
AFAIK, the Project crd shouldn't be available on k8s.
Is the purpose is to test the installation only? or also the invocation?
Or should only the workflows with script under e2e folder are tested for their execution?

Reply:
The purpose is to test only the installation
The workflows for which we want to test the execution, like M2K, are named `*-e2e` (for now only M2K, even MTA is not testing e2e, like in the serverless-workflow repo, I created an issue for that https://issues.redhat.com/browse/FLPATH-1830)